app.js

```diff
@@ -1,81 +1,83 @@
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
 
-if (localStorage.getItem('books') === null) {
-  storedBooks = localStorage.setItem('books', JSON.stringify(books));
-} else {
-  storedBooks = JSON.parse(localStorage.getItem('books'));
+  // remove book methods
+  removeBook(bookId) {
+    this.books = this.books.filter((item, index) => {
+      if (index !== bookId) {
+        return item;
+      }
+      return undefined;
+    });
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
 }
 
-const showBooks = () => {
-  storedBooks = JSON.parse(localStorage.getItem('books'));
-  storedBooks.forEach((book) => {
-    const div = document.createElement('div');
-    const header = document.createElement('h3');
-    header.textContent += book.title;
-    const authorName = document.createElement('p');
-    const button = document.createElement('button');
-    button.textContent = 'Remove';
-    button.className = 'remove';
-    div.appendChild(header);
-    div.appendChild(authorName);
-    div.appendChild(button);
-    displayBook.appendChild(div);
-  });
-};
+// create an object
+const listBooks = new Books();
+function onPageReload() {
+  displayBook.innerHTML = listBooks.books.map((book, index) => `
+  <div class="display">
+    <p>"${book.title}" by ${book.author}</p>
+    <button class="remove-btn" onclick="removeBook(${index})">Remove</button>
+    </div>
+  `).join('');
 
-const addBook = (event) => {
-  storedBooks = JSON.parse(localStorage.getItem('books'));
+  if (listBooks.books.length === 0) {
+    displayBook.style.cssText = 'border: none;';
+  } else {
+    displayBook.style.cssText = 'border: 3px solid black;';
+  }
+}
+
+form.addEventListener('submit', (event) => {
   event.preventDefault();
   const formData = new FormData(form);
   const title = formData.get('title');
   const author = formData.get('author');
-  const book = {
+  const bookObj = {
     title,
     author,
   };
-
-  book.id = Math.floor(Math.random() * (100 - 1 + 1)) + 1;
-  storedBooks.push(book);
-  localStorage.setItem('books', JSON.stringify(storedBooks));
+  listBooks.addBook(bookObj);
+  onPageReload();
   form.reset();
+});
 
-  const div = document.createElement('div');
-  div.classList.add('books');
-  div.dataset.id = book.id;
-  const header = document.createElement('h3');
-  header.textContent += book.title;
-  const authorName = document.createElement('p');
-  authorName.textContent += book.author;
-  const button = document.createElement('button');
-  button.textContent = 'Remove';
-  button.className = 'remove';
-  div.appendChild(header);
-  div.appendChild(authorName);
-  div.appendChild(button);
-  displayBook.appendChild(div);
-};
-
-const removeBook = (el) => {
-  if (el.classList.contains('remove')) {
-    el.parentElement.remove();
-  }
+/* eslint-disable no-unused-vars */
+const removeBook = (bookId) => {
+  listBooks.removeBook(bookId);
+  onPageReload();
 };
+/* eslint-disable no-unused-vars */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
Ok thanks we shall work on that